app/src/main/java/com/github/se/gomeet/ui/mainscreens/profile/Profile.kt

```diff
@@ -393,27 +396,43 @@ fun ProfileImage(
       contentScale = ContentScale.Crop)
 }
 
+fun generateQRCode(type: String, id: String): Bitmap {
```

Comment:
I think it would be better to implement this function not in the same file as a UI file, probably better to put this function in the `model` package or just somewhere else than in the `ui` package. Also don't forget about documentation

Reply:
Done